test/README.md

```diff
@@ -3,23 +3,38 @@
 
 These are tests for the WIP Resonate API (v4) migration. This iteration begins development of tests of the WIP api specification. This stage of the testing effort deliberately keeps things simple. Perhaps later iterations will focus on more fancy stuff.
 
-The purpose of these tests is to verify / validate the endpoints of this API, and to begin development of a more robust API test suite.
+The purpose of these tests is to verify / validate the endpoints of this API, and to begin development of a more robust API test suite. Integration / CICD tests will follow.
 
 These tests reference the [beam](https://github.com/resonatecoop/beam/tree/main/src/services) repository. The 'beam' repo serves as a reference for available endpoints, which are to be integrated into the api(v4) repo.
 
-The api(v4) tests live in the `/api/test` directory. In this directory there are three subfolders (`Admin.ts`, `Api.ts`, and `User.ts`), each of which corresponds to a file in the [beam](https://github.com/resonatecoop/beam/tree/main/src/services) repository.
+The api(v4) tests live in the `/api/test` directory. In this directory there are two subdirectories, `baseline` and `integration`. 
+* `baseline` tests cover the existing functionality. They depend on static, fixed test data and are intended to verify and validate the baseline functionality of the api, as this functionality is developed.
+  * In this directory there are three subfolders (`Admin.ts`, `Api.ts`, and `User.ts`), each of which corresponds to a file in the [beam](https://github.com/resonatecoop/beam/tree/main/src/services) repository.
+  * A list of the endpoints tested in `baseline` is [here](./ListOfBaselineEndpoints.md).
+* `integration` tests are intended for Github Actions and any other CI/CD needs. These test will be developed once the `baseline` test sets are stable.
+
+## How the test data was made
+You should never need to create test data. However, you can read an overview of how the test data was made [here](./HowTheTestDataWasMade.md). 
```

Comment:
```suggestion
You can read an overview of how the test data was made [here](./HowTheTestDataWasMade.md). 
```

I'd say this because I think people creating test data should actually be encouraged, as it's more resilient.

Reply:
> I have to disagree here. Opening the door to multiple test data sets creates the possibility of changing the test data and completely wrecking the tests.

(threading reply :))

It's not multiple test data sets, it's making sure that what every test stands on its own and doesn't affect other tests. By limiting what data exists for the _entire_ data set, we're limiting the tests we can do. Similarly, if one test tests whether we're creating data, that data _will_ show up in other tests as well if we're not cleaning up after ourselves for every test.

Reply:
Let's take this up after the baseline tests are working correctly.